examples/nirfmxbluetooth/acp-basic.py

```diff
@@ -314,6 +314,15 @@ def raise_if_error(response):
         print(f"Upper Relative Powers (dB)           : {upper_relative_power[i]}")
         print(f"Lower Margin (dB)                    : {lower_margin[i]}")
         print(f"Upper Margin (dB)                    : {upper_margin[i]}\n")
+except grpc.RpcError as rpc_error:
+    error_message = rpc_error.details()
+    if rpc_error.code() == grpc.StatusCode.UNAVAILABLE:
+        error_message = f"Failed to connect to server on {server_address}:{server_port}"
+    elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
+        error_message = (
+            "The operation is not implemented or is not supported/enabled in this service"
+        )
+    print(f"{error_message}")
```

Comment:
Should probably go to STDERR instead:
```python
    sys.stderr.write(f"{error_message}\n")
```
(Otherwise, note that we don't need to use an f-string just to print a str variable.)

Reply:
Ah, good catch. Updated to print to STDERR instead.